src/stan/services/experimental/advi/meanfield.hpp

```diff
@@ -0,0 +1,84 @@
+#ifndef STAN_SERVICES_EXPERIMENTAL_ADVI_MEANFIELD_HPP
+#define STAN_SERVICES_EXPERIMENTAL_ADVI_MEANFIELD_HPP
+
+#include <stan/callbacks/writer.hpp>
+#include <stan/services/util/experimental_message.hpp>
+#include <stan/services/util/initialize.hpp>
+#include <stan/services/util/rng.hpp>
+#include <stan/io/var_context.hpp>
+#include <stan/variational/advi.hpp>
+#include <boost/random/additive_combine.hpp>
+#include <string>
+#include <vector>
+
+namespace stan {
+  namespace services {
+    namespace experimental {
+      namespace advi {
+
```

Comment:
Missing doc for arguments... don't know the code well enough to write doc here...